acceptance-tests/upgrade/upgrade_suite_test.go

```diff
@@ -20,5 +26,29 @@ func init() {
 
 func TestUpgrade(t *testing.T) {
 	RegisterFailHandler(Fail)
+
+	releasedBrokerpakV140 = detectBrokerpakV140()
```

Comment:
In a Ginkgo test, this stuff is typically done in a `BeforeSuite()` hook. Although this works, using a hybrid of Go testing.T and Ginkgo can make things harder to understand.

Also, I would suggest making these functions because it makes it easier to reason about what's going on as you don't have to refer to global variables.